src/Service/Program.cs

```diff
@@ -15,6 +15,12 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.ApplicationInsights;
 
+#if NET8_0_OR_GREATER
```

Comment:
Why would you need .net framework specific condition? is this feature not available in lower version of .Net framework?

Reply:
The required packages for Otel had conflicts with other packages referenced in the .NET 6 version, so following @aaronpowell advice I've decided to focus on the .NET 8 version only.